README.md

```diff
@@ -244,7 +244,7 @@ Translations of the guide are available in the following languages:
   literals. Whichever one you pick - apply it consistently.
 
   As far as embedded expressions go, there are also two acceptable
-  options:
+  options
```

Comment:
I did not notice that. Must have hit backspace while reading that section. As you said in https://github.com/bbatsov/ruby-style-guide/pull/429#issuecomment-95178829 I will remove the section in general.